src/main/java/org/jabref/gui/preferences/PreferencesDialogView.java

```diff
@@ -52,6 +52,9 @@ public PreferencesDialogView(Class<? extends PreferencesTab> preferencesTabToSel
                   .load()
                   .setAsDialogPane(this);
 
+        getDialogPane().setPrefWidth(700);
+        getDialogPane().setPrefHeight(500);
```

Comment:
These numbers feel random, there is no comment why these numbers were chosen. How do they relate with the numbers in .fxml.